cmd/bb_runner/BUILD.bazel

```diff
@@ -65,6 +65,25 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+container_image(
+    name = "bb_runner_alpine_container",
+    base = "@alpine//image",
+    entrypoint = [
+        "/tini",
+        "-v",
+        "-g",
+        "--",
+        "/bb_runner",
+    ],
+    layers = [
+        ":bb_runner_layer",
+        ":passwd_layer",
+        ":tini_layer",
+    ],
+    user = "root",
```

Comment:
What can we do to keep this somewhat secure? Running build actions as root doesn't seem like a good idea. Can't we keep it running as a non-root user ID, but configure Docker/Kubernetes/.... to grant it `CAP_SYS_CHROOT`?